pages/docs/couple-your-code/couple-your-code-porting-v2-3.md

```diff
@@ -201,58 +208,67 @@ error: ‘class precice::SolverInterface’ has no member named ‘initializeDat
-
-```xml
-<mapping:rbf  ...>
-  <basis-function:... />
-</mapping:rbf>
-```
-
-Example:
-
-preCICE version 2 rbf configuration:
-
-```xml
-<mapping:compact-polynomial-c0 direction="read" from= ... support-radius="0.3" />
-```
-
-corresponding preCICE version 3 rbf configuration (using the recommended auto selection):
-
-```xml
-<mapping:rbf  direction="read" from= ...>
-  <basis-function:compact-polynomial-c0 support-radius="0.3" />
-</mapping:rbf>
-```
-
-A specific solver should only be configured if you want to force preCICE to use and stick to a certain solver, independent of your problem size and execution.
-
-- Renamed `<mapping:rbf... use-qr-decomposition="true" />` to `<mapping:rbf-global-direct ... > <basis-function:... /> </mapping:rbf-global-direct>`.
-- Remove all timings in the mapping configuration `<mapping: ... timing="initial/onadvance/ondemand" />`.
-- Remove the preallocations in the mapping configuration `<mapping: ... preallocation="tree/compute/estimate/save/off" />`.
-
-<!--
-- Add `<profiling mode="all" />` after the `<log>` tag if you need profiling data.
-- Replace `<export:vtk />` for parallel participants with `<export:vtu />` or `<export:vtp />`.
--->
-
-- Renamed the `<m2n: ... />` attributes `from` -> `acceptor` and `to` -> `connector`.
-
-- Moved and renamed the optional attribute `<read-data: ... waveform-order="1" />` to `<data:scalar/vector ... waveform-degree="1"`.
-
-- We dropped quite some functionality concerning [data actions](https://precice.org/configuration-action.html) as these were not used to the best of our knowledge and hard to maintain:
-  - Removed deprecated action timings `regular-prior`, `regular-post`, `on-exchange-prior`, and `on-exchange-post`.
-  - Removed action timings `read-mapping-prior`, `write-mapping-prior`, and `on-time-window-complete-post`.
-  - Removed `ComputeCurvatureAction` and `ScaleByDtAction` actions.
-  - Removed callback functions `vertexCallback` and `postAction` from `PythonAction` interface.
-  - Removed timewindowsize from the `performAction` signature of `PythonAction`. The new signature is `performAction(time, data)`
-
-- Replace `<min-iteration-convergence-measure min-iterations="3" ... />` by `<min-iterations value="3"/>`.
-
-- We removed the plain `Broyden` acceleration. You could use `IQN-IMVJ` instead, which is a [multi-vector Broyden variant](http://hdl.handle.net/2117/191193).
+- Rename the `<m2n: ... />` attributes `from` -> `acceptor` and `to` -> `connector`.
+- Add `<profiling mode="all" />` after the `<log>` tag if you need full profiling data.
```

Comment:
Trying to make this sound less like something one must/should do.
Maybe also link to the respective documentation page?
By the way, the "after the `<log>`" should not matter, right?

```suggestion
- You can now add `<profiling mode="all" />` after the `<log>` tag if you need full profiling data.
```